.circleci/config.yml

```diff
@@ -419,7 +419,7 @@ jobs:
     docker:
       - environment:
           *container_base_environment
-        image: ivoanjo/docker-library:ddtrace_rb_2_5_9
+        image: ghcr.io/datadog/dd-trace-rb/ruby:2.5.9-dd
```

Comment:
Minor: Since the repo name already includes `dd-trace-rb`, consider perhaps dropping the `-dd` suffix at the end?

Reply:
Fair enough! Consider documenting that somewhere ;)